src/client/const.h

```diff
@@ -545,7 +545,8 @@ namespace Otc
         GameSmoothWalkElevation = 115,
         GameNegativeOffset = 116,
         GameItemTooltipV8 = 117,
-        LastGameFeature = 118
+        GameForgeConvergence = 118,
+        LastGameFeature = 119
```

Comment:
You can leave it here without a number, so it will never be modified and will always be the last
```suggestion
        LastGameFeature,
```